### PR TITLE
[ Shared ] Update configuration docs for each platform

### PR DESCRIPTION
**Pull request by abevers**
Signed-off-by: abevers <arnoud.bevers@accenture.com>

**Changelog**
- Update `configuration/README.md` for each platform to correctly refer to our BAF build container
- Cleanup of the documentation in general - made it consistent across platforms with only specifics changing

**To be reviewed by**
@suvajit-sarkar 

**Linked issue**
#872

### CODE REVIEW

File: platforms/hyperledger-besu/configuration/README.md [hunk 1]
```diff
@@ -1,49 +1,49 @@
-# Playbooks
+# Hyperledger Besu Configuration
+This configuration folder contains Ansible playbooks and their corresponding roles, which are used to deploy Hyperledger Besu on Managed Kubernetes Cluster(s).
 
-## Getting started
-This folder contains ansible playbooks and their corresponding roles which are used to deploy Hyperledger Besu on Managed Kubernetes Cluster(s).
 
+## Approaches
+There are two approaches to deploy a DLT network using BAF: 
+- Use a dedicated instance to deploy and manage the DLT network. This is recommended for production environments, and requires some setup & manual installation of CLI tools. More info on setting up this dedicated instance is found [here](https://blockchain-automation-framework.readthedocs.io/en/latest/operations/configure_prerequisites.html#ansible-inventory-file).
```
Comment (suvajit-sarkar) [inline platforms/hyperledger-besu/configuration/README.md:7]: Try to rephrase the part "manual installation of CLI tools", just say manual setup of Ansible and other required libraries and tools for setting up the controller machine; not say instance say machine

File: platforms/hyperledger-besu/configuration/README.md [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
+- Use the 'BAF Build container' to create a containerized Ansible controller from which to deploy/manage your networks. This is recommended for development instances, as it is an easy way to build the required base environment for BAF deployment. More info can be found [here](https://blockchain-automation-framework.readthedocs.io/en/latest/developer/docker-build.html).
 
-## Prerequisites
+## Installation pre-requisites
+Some tools are required to run the BAF Automation, such as Kubernetes, Git (repository), Vault and more.
```
Comment (suvajit-sarkar) [inline platforms/hyperledger-besu/configuration/README.md:11]: Don't Some tools , rephrase to say BAF requires or makes use of

File: platforms/hyperledger-besu/configuration/README.md [hunk 3]
```diff
@@ -1,49 +1,49 @@
...
-5. A Git User with write access to all the branches in the chosen Git repository. This user should be a service user and not a federated user.
-6. Edited and saved network.yaml with all fields populated according to your requirements. A Sample network.yaml can be found [here](./samples/network-besu.yaml)
+## Configuration pre-requisites
+For each organization in the DLT network you need to set up the following:
+1. One Managed Kubernetes cluster; BAF is currently tested on Amazon EKS, which means you will need AWS CLI set up as well.
+2. A Hashicorp Vault installation for each organization which is initialized and unsealed. The Vault Address should be accessible from this machine (where the playbook is run) and the Kubernetes cluster. The Vault root token is used in the network configuration, so this should be available as well.
+3. A Git User with write access to all the branches in the chosen Git repository; as well as an access token.
+4. The network configuration file (`network.yaml`) which has been filled in according to your requirements. A sample `network.yaml` for Hyperledger Besu can be found [here](./samples/network-besu.yaml).
```
Comment (sownak) [inline platforms/hyperledger-besu/configuration/README.md:19]: "Sample networkyamls ..." and link to the samples directory only, not a specific file

File: platforms/hyperledger-besu/configuration/README.md (hunk 2 above)
Comment (abevers) [reply platforms/hyperledger-besu/configuration/README.md]: Resolved with commit [5f62fa1](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/5f62fa1c30d332957c60305b7555b9d8466594a9)

File: platforms/hyperledger-besu/configuration/README.md (hunk 1 above)
Comment (abevers) [reply platforms/hyperledger-besu/configuration/README.md]: Resolved with commit [5f62fa1](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/5f62fa1c30d332957c60305b7555b9d8466594a9)

File: platforms/hyperledger-besu/configuration/README.md [hunk 4]
```diff
@@ -1,49 +1,49 @@
...
 
 ## Execution 
-
-**NOTE**: If you are running the playbooks independently i.e. not using platforms/shared/configuration/site.yaml, then use the same network.yaml as input for each playbook.
-
 ### Step 1
-Ensure that the `network.yaml` is edited properly and saved. Follow the guidance in the comments and readme.
+Ensure that the `network.yaml` is edited properly and saved. Follow the guidance in the comments (from the sample file found [here](./samples/network-besu.yaml)).
```
Comment (sownak) [inline platforms/hyperledger-besu/configuration/README.md:23]: should point to readthedocs site not a specific sample file

File: platforms/hyperledger-besu/configuration/README.md [hunk 5]
```diff
@@ -1,49 +1,49 @@
...
+Execute the playbook by running the command below - executed from the root of the project:
 ```
-ansible-playbook platforms/shared/configuration/site.yaml -e "@./platforms/samples/network-besu.yaml"
+ansible-playbook platforms/shared/configuration/site.yaml -e "@/path/to/network-besu.yaml"
 ```
-Ensure that you have added an **SSH Key** with read-write permission to your git repository before executing the playbook. Follow steps in [README](../../shared/configuration/README.md).<br>
-If you just want to run this deploy-network.yaml playbook, then, from the *platforms/Hyperledger Besu/configuration* directory, run the following command for deploying the network
+The [platforms/shared/configuration/site.yaml](../../shared/configuration/site.yaml) is the master playbook which does basic environment setup, configures the Kubernetes cluster and then calls platform specific deployment playbooks.
+
+You can also only run the platform specific deployment playbooks by running the command below - executed from the root of the project:
```
Comment (sownak) [inline platforms/hyperledger-besu/configuration/README.md:32]: Add "after the prerequisites have been installed"

File: platforms/hyperledger-besu/configuration/README.md [hunk 6]
```diff
@@ -1,49 +1,49 @@
...
+After your Ansible command has completed. your nodes or the participants in the Hyperledger Besu network should be up and running. We are working on a verification document for Hyperledger Besu.
 
 ## Miscellaneous
 
-1. openssl.conf: This is the configuration file used to generate the Root CA certificates for Hyperledger Besu-CA.
+1. `./openssl.conf`: This is the configuration file used to generate the Root CA certificates for Besu-CA.
+
+2. If you want to reset the network, i.e. delete all created resources while setting up the Hyperledger Besu network, then run the following command from the root folder of the project:
+    ```
+    ansible-playbook platforms/hyperledger-besu/cleanup.yaml -e "@/path/to/network-besu.yaml"
```
Comment (sownak) [inline platforms/hyperledger-besu/configuration/README.md:46]: is it cleanup or reset-network? use site.yaml -e reset=true as well

**Review by sownak — CHANGES_REQUESTED** [review]

File: platforms/hyperledger-besu/configuration/README.md (hunk 6 above)
Comment (abevers) [reply platforms/hyperledger-besu/configuration/README.md:46]: For Besu specifically the file is called `cleanup.yaml`. I'll add the `-e "reset=true"` option as well, for all platforms.

Comment (abevers) [reply platforms/hyperledger-besu/configuration/README.md]: Resolved in commit [9f27d09](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/9f27d09fc1063e42936f0cdc81410dc63f7106e9)

File: platforms/hyperledger-besu/configuration/README.md (hunk 5 above)
Comment (abevers) [reply platforms/hyperledger-besu/configuration/README.md]: Resolved in commit [9f27d09](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/9f27d09fc1063e42936f0cdc81410dc63f7106e9)

File: platforms/hyperledger-besu/configuration/README.md (hunk 4 above)
Comment (abevers) [reply platforms/hyperledger-besu/configuration/README.md]: Resolved in commit [9f27d09](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/9f27d09fc1063e42936f0cdc81410dc63f7106e9)

File: platforms/hyperledger-besu/configuration/README.md (hunk 3 above)
Comment (abevers) [reply platforms/hyperledger-besu/configuration/README.md]: Resolved in commit [9f27d09](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/9f27d09fc1063e42936f0cdc81410dc63f7106e9)

File: platforms/hyperledger-indy/configuration/README.md [hunk 7]
```diff
@@ -1,45 +1,56 @@
...
+1. One Managed Kubernetes cluster; BAF is currently tested on Amazon EKS, which means you will need AWS CLI set up as well.
+2. A Hashicorp Vault installation for each organization which is initialized and unsealed. The Vault Address should be accessible from this machine (where the playbook is run) and the Kubernetes cluster. The Vault root token is used in the network configuration, so this should be available as well.
+3. A Git User with write access to all the branches in the chosen Git repository; as well as an access token.
+4. The network configuration file (`network.yaml`) which has been filled in according to your requirements. A sample `network.yaml` for Hyperledger Indy can be found in [this folder](./samples/).
 
-## Execution
-
-**NOTE**: If you are running the playbooks independently i.e. not using platforms/shared/configuration/site.yaml, then use the same network.yaml as input for each playbook.
+For other general pre-quisites, such as Docker images, Ambassador and DNS setup, please refer to the ['Configure Pre-requisites' guide](https://blockchain-automation-framework.readthedocs.io/en/latest/operations/configure_prerequisites.html).\
```
Comment (sownak) [inline platforms/hyperledger-indy/configuration/README.md:21]: remove \

File: platforms/r3-corda-ent/configuration/README.md [hunk 8]
```diff
@@ -1,64 +1,55 @@
...
 
-## Execution 
-
-**NOTE**: If you are running the playbooks independently i.e. not using platforms/shared/configuration/site.yaml, then use the same network.yaml as input for each playbook.
+For other general pre-quisites, such as Docker images, Ambassador and DNS setup, please refer to the ['Configure Pre-requisites' guide](https://blockchain-automation-framework.readthedocs.io/en/latest/operations/configure_prerequisites.html).
 
+## Execution 
 ### Step 1
-Ensure that the `network.yaml` is edited properly and saved. Follow the guidance in the comments and readme.
+Ensure that the `network.yaml` is edited properly and saved. For now, follow the guidance in the comment of the sample `network.yaml`(s); a comprehensive guide is WIP.
```
Comment (sownak) [inline platforms/r3-corda-ent/configuration/README.md:25]: Is not WIP

**Review by sownak — CHANGES_REQUESTED** [review]

File: platforms/hyperledger-indy/configuration/README.md (hunk 7 above)
Comment (abevers) [reply platforms/hyperledger-indy/configuration/README.md]: Resolved in [a9c1489](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/a9c14893b67ea6f9eb1ef2e8a011949764891e53)

File: platforms/r3-corda-ent/configuration/README.md (hunk 8 above)
Comment (abevers) [reply platforms/r3-corda-ent/configuration/README.md]: True, wording mistake. Resolved in [a9c1489](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/a9c14893b67ea6f9eb1ef2e8a011949764891e53)

Comment (sownak) [reply platforms/r3-corda-ent/configuration/README.md]: All corda enterprise details are there in the existing cordanetworkyaml in docs. So give the same link as done for corda opensource.

Comment (abevers) [reply platforms/r3-corda-ent/configuration/README.md]: You're correct. The Corda OS documentation has extra distinction for Enterprise. I'll link to this. For future, should we create a seperate page for Corda Enterprise?

Comment (sownak) [reply platforms/r3-corda-ent/configuration/README.md]: No, they are the same platform, so no separate operational pages should be created.

Comment (abevers) [reply platforms/r3-corda-ent/configuration/README.md]: Resolved in [f95b02a](https://github.com/hyperledger-labs/blockchain-automation-framework/commit/f95b02a3224e22529f3b2556af2b8962c68d78d9)

**Review by sownak — APPROVED** [review]

**Pull request merged**